yrs/src/update.rs

```diff
@@ -336,17 +337,22 @@ impl Update {
         client_block_ref_ids: &'a mut Vec<ClientID>,
         blocks: &'b mut UpdateBlocks,
     ) -> Option<(ClientID, &'b mut VecDeque<BlockCarrier>)> {
-        loop {
-            if let Some((id, Some(client_blocks))) = client_block_ref_ids
```

Comment:
This case was too restrictive - if we returned `Some((id, None))` (meaning current block store didn't have a client id we were looking for), then all subsequent blocks from the remaining clients waiting in update were dropped.